doc/modules/classes.rst

```diff
@@ -971,6 +971,25 @@ See the :ref:`metrics` section of the user guide for further details.
     multiclass.OneVsOneClassifier
     multiclass.OutputCodeClassifier
 
+.. _multioutput_ref:
+
+:mod:`sklearn.multioutput`: Multioutput regression
+==================================================
+
+.. automodule:: sklearn.multioutput
+   :no-members:
+   :no-inherited-members:
+
+**User guide:** See the :ref:`multiclass` section for further details.
```

Comment:
The narrative docs are in the same section as the multiclass stuff and the ref in the docs is just `multiclass`